src/placeos-driver/proxy/remote_driver.cr

```diff
@@ -182,7 +182,7 @@ module PlaceOS::Driver::Proxy
 
       exec_args = args || named_args
 
-      client = Core::Client.new(which_core)
+      client = Core::Client.client(which_core, request_id)
       client.execute(module_id, function, exec_args)
```

Comment:
This creates the client, and leaves it open. Could you use [this method?](https://github.com/PlaceOS/core/blob/master/src/placeos-core/client.cr#L34)